release fails at deploy step - no distributionManagement

**Describe the bug**

Running the release with command:

```
./release.sh -k 'zzzzz' -a 0.3.0
```

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-deploy-plugin:3.1.1:deploy (default-deploy) on project kroxylicious-bom: Deployment failed: repository element was not specified in the POM inside distributionManagement element or in -DaltDeploymentRepository=id::url parameter -> [Help 1]
[ERROR]
```

The issue is there is no distributionManagement declared with the BOM.

**To Reproduce**
Steps to reproduce the behavior:
1. Configure kroxylicious like this ...
2. Run command '....'
3. See error

**Expected behavior**
A clear and concise description of what you expected to happen.

**YAML files and logs**

Attach or copy paste your kroxylicious configuration YAML (with sensitive details
like broker address redacted).
Attach or copy and paste relevant logs.

**Additional context**
Add any other context about the problem here.

--- a/kroxylicious-bom/pom.xml
+++ b/kroxylicious-bom/pom.xml
@@ -80,6 +80,17 @@
         <url>https://github.com/kroxylicious/kroxylicious</url>
     </scm>
 
+    <distributionManagement>
+        <snapshotRepository>
+            <id>ossrh</id>
+            <url>https://s01.oss.sonatype.org/content/repositories/snapshots</url>
+        </snapshotRepository>
+        <repository>
+            <id>ossrh</id>
+            <url>https://s01.oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+        </repository>
+    </distributionManagement>
+
     <dependencyManagement>
         <dependencies>
             <!-- Core Production dependencies -->

